src/minishell.c

```diff
@@ -26,6 +26,15 @@ static void	print_envp(char *envp[])
 }
 */
 
+static void	sighandler(int signal)
+{
+	(void)signal;
+	printf("\n");
+	rl_replace_line("", 0);
```

Comment:
I'm working on Linux at home right now, so let's see how it connects to the new library.